docs/administration-guide/ssh-tunnel-for-database-connections.md

```diff
@@ -0,0 +1,75 @@
+
+### How to use this feature
+
+When connecting though a bastion host:
+
+- Answer yes to the "Use an SSH-tunnel for database connections" parameter
+- Enter the hostname for the data warehouse as it is seen from inside the network in the `Host` parameter.
+- Enter the data warehouse port as seen from inside the network into the `Port` parameter.
+- Enter the external name of the bastion host as seen from the outside of the network (or wherever you are) into the `SSH tunnel host` parameter.
+- Enter the ssh port as seen from outside the network into the `SSH tunnel port` parameter. This is usually 22, regardless of which data warehouse you are connecting to.
+- Enter the username and password you use to login to the bastion host into the `SSH tunnel username` and `SSH tunnel password` parameters.
+
+If you are unable to connect test your ssh credentials by connecting to the SSH server/Bastion Host using ssh directly:
+
+```
+ssh <SSH tunnel username>@<SSH tunnel host> -p <SSH tunnel port>
+```
+
+Another common case where direct connections are not possible is when connecting to a data warehouse that is only accessible locally and does not allow remote connections. In this case you will be opening an SSH connection to the data warehouse, then from there connecting back to the same computer.
+
+- Answer yes to the "Use an SSH-tunnel for database connections" parameter
+- Enter `localhost` in the `Host` parameter. This is the name the server
+- Enter the same value in the `Port` parameter that you would use if you where sitting directly at the data warehouse host system.
+- Enter the extenal name of the data warehouse, as seen from the outside of the network (or wherever you are) into the `SSH tunnel host` parameter.
+- Enter the SSH port as seen from outside the network into the `SSH tunnel port` parameter. This is usually 22, regardless of which data warehouse you are connecting to.
+- Enter the username and password you use to login to the bastion host into the `SSH tunnel username` and `SSH tunnel password` parameters.
+
+If you have problems connecting, verify the SSH host port and password by connecting manually using ssh or PuTTY on older windows systems.
+
+### Disadvantages to indirect connections
+
+While using an ssh tunnel makes it possible to use a data warehouse that is otherwise inaccessible, it is almost always preferable to use a direct connection when possible.
```

Comment:
```suggestion
While using an SSH tunnel makes it possible to use a data warehouse that is otherwise inaccessible, it is almost always preferable to use a direct connection when possible.
```